src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2066,7 +2078,8 @@ export default class Aircraft {
                 }
 
                 this.target.speed = crange(3, offset[1], 10, this.model.speed.landing, this.fms.currentWaypoint().start_speed);
-            } else if ((this.altitude - runway_elevation) >= 300 && this.mode === FLIGHT_MODES.LANDING) {
+            }
```

Comment:
This gives me the heebiejeebies, just because it's dramatically different, but I could see how it could be much easier to work with. #inNateWeTrust